src/global-css.ts

```diff
@@ -79,7 +79,7 @@ import "ui/components/shared/SettingsModal/SettingsModal.css";
 import "ui/components/shared/SettingsModal/SettingsNavigation.css";
 import "ui/components/shared/SharingModal/CollaboratorsList.css";
 import "ui/components/shared/SharingModal/EmailForm.css";
-import "ui/components/shared/SharingModal/ReplayLink.css";
+import "ui/components/shared/SharingModal/SharingModal.module.css";
```

Comment:
CSS modules should only be imported by components, e.g.

```tsx
// MyComponent.tsx
import styles from "MyComponent.module.css";
```

Importing one from a global style sheet like this has no impact (because the style names are all mangled and unused.)

```suggestion
```